providers/common/der/der_slh_dsa_key.c

```diff
@@ -0,0 +1,42 @@
+/*
+ * SLH-DSA low level APIs are deprecated for public use, but still ok for
```

Comment:
Stating APIs we have never exposed are "deprecated" seems odd - I know what you mean - but the words aren't quite right.

Reply:
Removed and put copyright in..